css/style.css

```diff
@@ -97,28 +97,65 @@ footer li {
   list-style-type: none;
   padding: 0 10px;
 }
+.issues li {
+  position: relative;
+  counter-increment: issue-counter;
+}
+
 .issues li a {
   text-decoration: none;
 }
+
+.issues h6 {
+  font-size: 1.8rem;
+}
+
+.issues-title {
+
+  font-size: 4rem;
```

Comment:
The title looks slightly too big, to me at least. What about deleting this `font-size: 4rem;` so that the Issue title reverts to \<h3\>'s default size of 34px? 34px also matches the [title for each row of projects](https://github.com/code4sac/codeforsacramento.org/blob/gh-pages/projects/index.html#L41) on the page behind this modal, so it stays consistent with that.

Reply:
Sounds good, I removed that styling.